hedera-node/src/main/java/com/hedera/services/context/primitives/StateView.java

```diff
@@ -443,7 +443,8 @@ public boolean scheduleExists(ScheduleID id) {
 				.setAutoRenewPeriod(Duration.newBuilder().setSeconds(account.getAutoRenewSecs()))
 				.setBalance(account.getBalance())
 				.setExpirationTime(Timestamp.newBuilder().setSeconds(account.getExpiry()))
-				.setContractAccountID(asSolidityAddressHex(id));
+				.setContractAccountID(asSolidityAddressHex(id))
+				.setOwnedNfts(account.getNFTsOwned());
```

Comment:
Do we always have this owner flag to set. What if we are dealing with fungible and not nft?

Reply:
This is a query that returns information for a given account. Its not related to tokens.